check_springboot_actuator.py

```diff
@@ -143,10 +146,13 @@ def handle_version_2():
         details = json_data
     if version == 2:
         details = json_data['status']
+    if version == 3:
+        details = json_data['components']
```

Comment:
We have some healthpoints with no components (only status = up). if there no components, the script will fail with `keyerror`. our soulution is following... maybe some better ideas there

`        details = json_data.get('components') or []`